config/crd/bases/operator.etcd.io_etcdclusters.yaml

```
@@ -39,6 +39,13 @@ spec:
           spec:
             description: EtcdClusterSpec defines the desired state of EtcdCluster.
             properties:
+              etcdOptions:
+                description: etcd configuration options are passed as environmental
+                  vars to the etcd container, refer to etcd documentation for configuration
+                  options application for the version of etcd being used.
+                items:
+                  type: string
+                type: array
```

Nit: Minor wording tweak to align with direction

```suggestion
              etcdOptions:
                description: etcd configuration options are passed as command line
                 arguments to the etcd container, refer to etcd documentation for 
                 configuration options applicable for the version of etcd being used.
                items:
                  type: string
                type: array
```

---

Thanks for the pickup, made the updates